dropwizard-testing/src/test/java/io/dropwizard/testing/app/OverrideDefaultExceptionMapperTest.java

```diff
@@ -0,0 +1,42 @@
+package io.dropwizard.testing.app;
+
+import io.dropwizard.testing.junit.ResourceTestRule;
+import org.junit.ClassRule;
+import org.junit.Test;
+
+import javax.validation.ConstraintViolationException;
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.ExceptionMapper;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+
+/**
+ * Test to make sure that the default exception mappers registered in
+ * {@link ResourceTestRule} can be overridden
+ */
+public class OverrideDefaultExceptionMapperTest {
+    private static final PeopleStore dao = mock(PeopleStore.class);
+
+    @ClassRule
+    public static final ResourceTestRule resources = ResourceTestRule.builder()
+            .addResource(new PersonResource(dao))
+            .addProvider(new CustomMapper())
+            .build();
+
+    private static class CustomMapper implements ExceptionMapper<ConstraintViolationException> {
```

Comment:
Please, move the class to the start or to the end of `OverrideDefaultExceptionMapperTest`, so it's more easy to track.